### PR TITLE
[cometvisu] fix cometvisu dataprovider response values

### PR DESCRIPTION
**Pull request by peuter**
This PR fixes several errors in the dataprovider responses used by the CometVisu editor. 
The main goal is to fix the listing of available addresses (aka Item-Names) in the editor.

It also handles some changes in the data-structure of the current development version of the CometVisu and fixes the responses for the listing of items, designs and plugins.

### CODE REVIEW

File: addons/ui/org.openhab.ui.cometvisu/src/main/java/org/openhab/ui/cometvisu/servlet/CometVisuServlet.java [hunk 1]
```diff
@@ -875,42 +903,92 @@ private final void dataProviderService(File file, HttpServletRequest request, Ht
...
+                } catch (XPathExpressionException e) {
+                    logger.error("error extracting items from SVG file: {}", e);
+                }
+            } else {
+                File iconDir = new File(resourceFolder, "icon/knx-uf-iconset/128x128_white/");
+                if (iconDir.exists() && iconDir.isDirectory()) {
+                    FilenameFilter filter = new FilenameFilter() {
+                        @Override
+                        public boolean accept(File dir, String name) {
+                            // TODO Auto-generated method stub
```
Comment (kaikreuzer) [inline addons/ui/org.openhab.ui.cometvisu/src/main/java/org/openhab/ui/cometvisu/servlet/CometVisuServlet.java]: can be removed

File: addons/ui/org.openhab.ui.cometvisu/src/main/java/org/openhab/ui/cometvisu/servlet/CometVisuServlet.java [hunk 2]
```diff
@@ -875,42 +903,92 @@ private final void dataProviderService(File file, HttpServletRequest request, Ht
...
+                        Node node = nl.item(i);
                         DataBean bean = new DataBean();
-                        bean.label = iconName;
-                        bean.value = iconName;
+                        bean.label = node.getTextContent();
+                        bean.value = node.getTextContent();
                         beans.add(bean);
                     }
+                } catch (SAXException e) {
+                    logger.error("error parsing SVG file: {}", e);
```
Comment (kaikreuzer) [inline addons/ui/org.openhab.ui.cometvisu/src/main/java/org/openhab/ui/cometvisu/servlet/CometVisuServlet.java]: Change to 
```
logger.error("error parsing SVG file: {}", e.getMessage(), e);
```

File: addons/ui/org.openhab.ui.cometvisu/src/main/java/org/openhab/ui/cometvisu/servlet/CometVisuServlet.java [hunk 3]
```diff
@@ -875,42 +903,92 @@ private final void dataProviderService(File file, HttpServletRequest request, Ht
...
+                } catch (ParserConfigurationException e) {
+                    logger.error("error extracting items from SVG file: {}", e);
```
Comment (kaikreuzer) [inline addons/ui/org.openhab.ui.cometvisu/src/main/java/org/openhab/ui/cometvisu/servlet/CometVisuServlet.java]: dito

File: addons/ui/org.openhab.ui.cometvisu/src/main/java/org/openhab/ui/cometvisu/servlet/CometVisuServlet.java [hunk 4]
```diff
@@ -875,42 +903,92 @@ private final void dataProviderService(File file, HttpServletRequest request, Ht
...
+                } catch (XPathExpressionException e) {
+                    logger.error("error extracting items from SVG file: {}", e);
```
Comment (kaikreuzer) [inline addons/ui/org.openhab.ui.cometvisu/src/main/java/org/openhab/ui/cometvisu/servlet/CometVisuServlet.java]: dito

**Comment by peuter** [comment]
Ok, done.

**Comment by kaikreuzer** [comment]
Thanks!

**Pull request merged**